CIP-0054/README.md

```diff
@@ -0,0 +1,225 @@
+---
+CIP: 54
+Title: Cardano Smart NFTs
+Authors: Kieran Simkin
+Comments-URI: https://forum.cardano.org/t/cip-draft-cardano-smart-nfts/100470
+Status: Draft
+Type: Standards
```

Comment:
This one would be a "Process" track as it does not entail any changes in the _core_ Cardano components (no ledger or consensus changes; it is purely built upon the current system capabilities).

Reply:
Support of the mutable NFT part is necessary for recording system eternal changes and will allow to chase use cases like gaming (build your character, add items to stash, work your stats) as well as multi-verse cases (transfer your character and play in another game; interactions with multi-verse market places, etc.). You can also chase cases as Cardano Weather Station, Cardano Seismic Activity, Cardano Financial Markets, New Google Maps layers from Cardano NFTs (Helium internet coverage, Favourite fruit prices in your surroundings,...). And many many many more!